lib/util.js

```diff
@@ -10,19 +10,20 @@ Rye.define('Util', function(){
         }
 
     function each (obj, fn, context) {
+        var i = null
```

Comment:
We don't really follow crock style vars-at-the-top. It mandates a linter to avoid mistakes.